doc/releases/release-notes-2.4.rst

```diff
@@ -409,6 +416,11 @@ Networking
 * Fixed LWM2M Security and Server object instance matching.
 * Fixed LWM2M updating lifetime on Register Update event.
 * Fixed MQTT double CONNACK event notification on server reject.
+* PPP: Added initial support for PAP authentication.
+* PPP: Automatic select CONFIG_RING_BUFFER as it is required.
```

Comment:
IMHO no need to mention this in the release note. I went through all the changes and only picked relevant stuff and not all changes. It seems I missed some important stuff like the PAP support.